### PR TITLE
Renamed some details and added dropdown for pull image

### LINKED ISSUE #101: ZUI: show potential commands to pull the image:tag

### LINKED ISSUE #109: ZUI: "Weekly Downloads" is actually "Total Downloads" for the entire life of the image

### PR DESCRIPTION
**Pull request by ameliamariabreda**
**What type of PR is this?**
cleanup
feature 

**Which issue does this PR fix**:
Closes #101 
Closes #109 

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs from IPAMD/CNI showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades. Has updating a running cluster been tested?**:

**Does this change require updates to the CNI daemonset config files to work?**:

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/__tests__/RepoPage/Repo.test.js [hunk 1]
```diff
@@ -82,6 +82,6 @@ describe('Repo details component', () => {
     jest.spyOn(api, 'get').mockResolvedValue({ status: 200, data: { data: mockRepoDetailsData } });
     render(<RepoDetails />);
     fireEvent.click(await screen.findByTestId('pullcopy-btn'));
-    await waitFor(() => expect(mockCopyToClipboard).toHaveBeenCalledWith('Pull test'));
+    await waitFor(() => expect(mockCopyToClipboard).toHaveBeenCalledWith('docker pull http://localhost/test'));
```
Comment (andaaron) [inline src/__tests__/RepoPage/Repo.test.js:85]: This needs to have multiple tabs, and the user can select docker/podman/skopeo.

For docker: `docker pull 127.0.0.1:5000/alpine:latest`
For podman: `podman pull 127.0.0.1:5000/alpine:latest`
For skopeo: `skopeo copy docker://127.0.0.1:5000/alpine:latest docker://artprod.mycompany:port/<DOCKER_REPOSITORY>:<DOCKER_TAG>`

Where `alpine`, `latest` are information on this specific repo/tag (this feature should be available after you select an image:tag). Also the source where you copy from should not be hardcoded, you can probably get the hostname from some configuration (it's the same hostname where the backend resides).

File: src/components/RepoDetails.jsx [hunk 2]
```diff
@@ -307,32 +312,40 @@ function RepoDetails() {
...
+                      sx={{ m: 1, width: '20.625rem', borderRadius: '0.5rem', color: '#14191F', alignContent: 'left' }}
+                    >
+                      <MenuItem value={`docker pull ${host()}/${name}`}>
+                        docker pull {host()}/{name}
+                      </MenuItem>
+                      <MenuItem value={`podman pull ${host()}/${name}`}>
+                        podman pull {host()}/{name}
+                      </MenuItem>
+                      <MenuItem value={`skopeo pull ${host()}/${name}`}>
+                        skopeo pull {host()}/{name}
```
Comment (raulkele) [inline src/components/RepoDetails.jsx:346]: The skopeo command is different, check Andrei's comment

File: src/__tests__/RepoPage/Repo.test.js (hunk 1 above)
Comment (raulkele) [reply src/__tests__/RepoPage/Repo.test.js:85]: This is the test file, the logic is (currently) in the RepoDetails.jsx file

File: src/components/RepoDetails.jsx [hunk 3]
```diff
@@ -307,32 +312,40 @@ function RepoDetails() {
                   </Stack>
                 </Grid>
                 <Grid item xs={4}>
-                  <Typography variant="body1" sx={{ color: '#52637A', fontSize: '1rem' }}>
-                    Copy and pull to pull this image
-                  </Typography>
+                  <Stack direction="row">
```
Comment (raulkele) [inline src/components/RepoDetails.jsx:315]: The pull command should be in the TagDetails page as opposed to the RepoDetails page. We also need to forward this comment to UX so they can update the mocks accordingly.

**Review by raulkele — CHANGES_REQUESTED** [review]

File: src/__tests__/RepoPage/Repo.test.js (hunk 1 above)
Comment (andaaron) [reply src/__tests__/RepoPage/Repo.test.js:85]: Ok, I see. nevermind then :)

**Review by raulkele — APPROVED** [review]

**Pull request merged**